### PR TITLE
Give console app template name + description

### PR DESCRIPTION
**Pull request by davkean**

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/project-system @natidea

**Comment by 333fred** [comment]
:+1:

**Comment by srivatsn** [comment]
So how would the web template get its description?

**Comment by srivatsn** [comment]
Or desktop console app? I mean it sounds odd for the project system itself to have a string about .NET Core console apps.

**Comment by davkean** [comment]
@333fred Can you look at the test failure?

**Comment by davkean** [comment]
To have resources, you need a package. Given recent push to consolidate dlls and avoid unnecessary dlls where you would put this instead?

**Comment by srivatsn** [comment]
Oh I see so the template specifies Package=x, Id = # 3. This is just a parking spot as opposed to any special casing in the project system. Fine by me then. 👍

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/InterceptedProjectProperties/LocalDebuggerCommandArgumentsProviderTests.cs [hunk 1]
```diff
@@ -58,15 +58,14 @@ public void LocalDebuggerCommandArgumentsProvider_NullAsConfiguredProject_Throws
         }
 
         [Theory]
-        [InlineData("consoleapp1.exe", "consoleapp1", false, "arg1 arg2")]
-        [InlineData("consoleapp1.exe", "consoleapp1", true, "arg1 arg2")]
-        public async Task LocalDebuggerCommandArgumentsProvider_EmptyCommand_ReturnsExec(string executable, string dll, bool useOutputGroups, string args)
+        [InlineData(@"C:\Temp\consoleapp1.exe", "arg1 arg2", false,  @"exec C:\Temp\consoleapp1.dll arg1 arg2")]
```
Comment (333fred) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/InterceptedProjectProperties/LocalDebuggerCommandArgumentsProviderTests.cs]: Missing the quotes around the binary path. Also, can you remove the unused System.IO using at the top?

Comment (333fred) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/InterceptedProjectProperties/LocalDebuggerCommandArgumentsProviderTests.cs]: Should this also be a separate pull?

**Pull request merged**